.eslintrc.js

```diff
@@ -1,9 +1,4 @@
 module.exports = {
-  root: true,
```

Comment:
Why did you remove that part of the ESLint config? :thinking:

Reply:
Other than that, LGTM.

Reply:
You're right, I have to set this on the other modules too! damn it

Reply:
What's with

`parserOptions: {
    parser: 'babel-eslint',
    sourceType: 'module'
  },` :thinking: 	

Using babel-eslint is crucial in case the module is using ES proposals that are not part of the language yet.

Reply:
In this module there is nothing that needs `babel-eslint`

Reply:
Yes, not in this module :relaxed: 	I meant with respect to other modules

Reply:
Ah yes, in some it is necessary